<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BandPowerSettingsViewWidget</class>
 <widget class="QWidget" name="BandPowerSettingsViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>176</width>
       <height>240</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" rowspan="2" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxMin">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string> Hz</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSlider" name="horizontalSliderMin">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="radioButtonAR">
       <property name="text">
        <string>AR</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupMethod</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Detrend</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Maximum</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QRadioButton" name="radioButtonFFT">
       <property name="text">
        <string>FFT</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupMethod</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Minimum</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxMax">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string> Hz</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Intervall Length</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSlider" name="horizontalSliderMax">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBoxIntervallLength">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string> x</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Method</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QRadioButton" name="radioButtonMean">
       <property name="text">
        <string>Mean</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupDetrend</string>
       </attribute>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QRadioButton" name="radioButtonNoDetrend">
       <property name="text">
        <string>None</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupDetrend</string>
       </attribute>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QRadioButton" name="radioButtonLinear">
       <property name="text">
        <string>Linear</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupDetrend</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupMethod"/>
  <buttongroup name="buttonGroupDetrend"/>
 </buttongroups>
</ui>
